config/software/datadog-agent.rb

```diff
@@ -46,7 +46,12 @@
         copy 'packaging/debian/start_agent.sh', '/opt/datadog-agent/bin/start_agent.sh'
         command 'chmod 755 /opt/datadog-agent/bin/start_agent.sh'
       end
-      copy 'packaging/supervisor.conf', '/etc/dd-agent/supervisor.conf'
+      # Use a supervisor conf with go-metro on 64-bit platforms only
+      if ohai['kernel']['machine'] == 'x86_64'
+        copy 'packaging/supervisor.conf', '/etc/dd-agent/supervisor.conf'
+      else
+        copy 'packaging/supervisor_wo_metro.conf', '/etc/dd-agent/supervisor.conf'
```

Comment:
if you decide to change the name of `supervisor_wo_metro.conf` to `supervisor_32.conf` - update here accordingly.